setup.cfg

```diff
@@ -33,7 +33,7 @@ install_requires =
 [options.extras_require]
 tests =
     check-manifest>=0.49
-    pytest-black>=0.3.0
+    pytest-black-ng>=0.3.0
```

Comment:
comment/shelve: abandoned libraries strike again... looks good for now, and since this isn't an Invenio module we can't depend on `pytest-invenio` anyways.

We should definitely look into https://github.com/inveniosoftware/pytest-invenio/pull/99 to centralize the dependency (since it looks like it has to be changed across all Invenio modules now?) or as a more "universal" alternative integrate it [into GitHub Actions](https://black.readthedocs.io/en/stable/integrations/github_actions.html) (though it would be even better to have centralized GitHub Actions workflows as we discussed in other places).

Reply:
yes, but the reusable workflows doesn't solve the problem with the local tests. so if i can't install all my dependencies to test the package locally over the setup.cfg file i have to install them separately which is in my thinking annoying.

but nevertheless a huge GO for the reusable workflows. as pointed out somewhere else tu-graz-library has been using his approach for almost a year. 

we should combine both approaches. centralize in pytest-invenio and use reusable workflows